src/Symfony/Component/HttpKernel/Attribute/MapQueryParameter.php

```diff
@@ -32,6 +34,7 @@ public function __construct(
         public int $flags = 0,
         public array $options = [],
         string $resolver = QueryParameterValueResolver::class,
+        public readonly int $validationFailedStatusCode = Response::HTTP_NOT_FOUND,
```

Comment:
I'd remove readonly for consistency with the other properties. It wouldn't provide any benefits anyway in practice.